docs/pages/application-access/getting-started.mdx

```diff
@@ -62,8 +62,10 @@ $ tctl tokens add \
 
 ### Start Teleport
 
-On the host where you will run the Teleport Application Service, download the latest version of Teleport for your platform from our
-[downloads page](https://goteleport.com/teleport/download).
+
+Install Teleport on the host where you will run the Teleport Application Service. See our [Installation](/docs/installation/) page for options besides Linux servers.
```

Comment:
The `Installation` link should lead to an .mdx page so our NextJS tooling can fill in the `/ver/<version>` path segments in the URL as needed.

Reply:
Also looks like there are some outstanding suggestions.

Reply:
addressed